Assets/LeapMotionModules/PinchUtility/Scripts/LeapPinchDetector.cs

```diff
@@ -157,6 +160,7 @@ public class LeapPinchDetector : MonoBehaviour {
 
       float pinchDistance = hand.PinchDistance * MM_TO_M;
       transform.rotation = hand.Basis.Rotation();
+      transform.position = hand.PalmPosition.ToVector3();
```

Comment:
This line does nothing, as the position is assigned to a couple lines below.  The transform is already correctly being assigned, so there doesn't need to be anything additional added here.